modules/table/src/main/java/org/apache/ignite/internal/table/distributed/index/IndexBuildTask.java

```diff
@@ -87,6 +90,7 @@ class IndexBuildTask {
         this.busyLock = busyLock;
         this.batchSize = batchSize;
         this.node = node;
+        this.listeners = listeners;
```

Comment:
Is it intended that `listeners` is live (that is, the list sees changes made in `IndexBuilder`)?

Reply:
Add comment in code.